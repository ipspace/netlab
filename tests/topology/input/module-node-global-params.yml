#
# Module merge test: node-only modules
#
---
defaults:
  device: iosv

module: [ ospf ]
bgp.as: 65000

nodes:
  r1:
  r2:
    module: [ bgp ]
