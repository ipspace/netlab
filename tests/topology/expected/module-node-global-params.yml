bgp:
  advertise_loopback: true
  as: 65000
  community:
    ebgp:
    - standard
    ibgp:
    - standard
    - extended
  next_hop_self: true
groups:
  as65000:
    members:
    - r2
input:
- topology/input/module-node-global-params.yml
- package:topology-defaults.yml
links:
- interfaces:
  - ifindex: 1
    ifname: GigabitEthernet0/1
    ipv4: 10.1.0.1/30
    node: r1
  - ifindex: 1
    ifname: GigabitEthernet0/1
    ipv4: 10.1.0.2/30
    node: r2
  linkindex: 1
  node_count: 2
  prefix:
    ipv4: 10.1.0.0/30
  type: p2p
module:
- bgp
- ospf
name: input
nodes:
  r1:
    af:
      ipv4: true
    box: cisco/iosv
    device: iosv
    id: 1
    interfaces:
    - ifindex: 1
      ifname: GigabitEthernet0/1
      ipv4: 10.1.0.1/30
      linkindex: 1
      name: r1 -> r2
      neighbors:
      - ifname: GigabitEthernet0/1
        ipv4: 10.1.0.2/30
        node: r2
      ospf:
        area: 0.0.0.0
        network_type: point-to-point
        passive: false
      type: p2p
    loopback:
      ipv4: 10.0.0.1/32
    mgmt:
      ifname: GigabitEthernet0/0
      ipv4: 192.168.121.101
      mac: 08:4f:a9:00:00:01
    module:
    - ospf
    name: r1
    ospf:
      af:
        ipv4: true
      area: 0.0.0.0
      router_id: 10.0.0.1
  r2:
    af:
      ipv4: true
    bgp:
      advertise_loopback: true
      as: 65000
      community:
        ebgp:
        - standard
        ibgp:
        - standard
        - extended
      neighbors: []
      next_hop_self: true
      router_id: 10.0.0.2
    box: cisco/iosv
    device: iosv
    id: 2
    interfaces:
    - ifindex: 1
      ifname: GigabitEthernet0/1
      ipv4: 10.1.0.2/30
      linkindex: 1
      name: r2 -> r1
      neighbors:
      - ifname: GigabitEthernet0/1
        ipv4: 10.1.0.1/30
        node: r1
      type: p2p
    loopback:
      ipv4: 10.0.0.2/32
    mgmt:
      ifname: GigabitEthernet0/0
      ipv4: 192.168.121.102
      mac: 08:4f:a9:00:00:02
    module:
    - bgp
    name: r2
ospf:
  area: 0.0.0.0
provider: libvirt
